ROADMAP.md

```diff
@@ -0,0 +1,58 @@
+# go-libp2p roadmap Q4’22/Q1’23
+
+## 📊 Comprehensive Metrics
+
+Why: For far too long, go-libp2p has been a black box. This has hurt us many times, by allowing trivial bugs to go undetected for a long time ([example](https://github.com/ipfs/kubo/pull/8750)). Having metrics will allow us to track the impact of performance improvements we make over time.
+
+Goal: go-libp2p ships with Grafana dashboards that allow node operators to monitor their nodes in production.
```

Comment:
```suggestion
Goal:  Export a wider set of metrics across go-libp2p components and enable node operators to monitor their nodes in production. Optionally provide a sample Grafana dashboards similar to the [rust-libp2p dashboard](https://kademlia-exporter.max-inden.de/d/Pfr0Fj6Mk/rust-libp2p?orgId=1&refresh=30s)
```

Reply:
Resolved by pointing to our rcmgr Grafana dashborad.